packages/@icon-magic/config-reader/src/schemas/config-schema.ts

```diff
@@ -67,6 +67,16 @@ const generateConfigProperties = {
 const topLevelConfigProperties = {
   iconPath: { type: 'string', minLength: 1 },
   iconName: { type: ['string', 'null'] },
+  iconLabels : {
+    type: 'array',
+    items: {
+      type: 'string'
+    }
+  },
+  iconCategory: {
+    type: 'string',
+    enum: ['background.svg', 'icons.svg']
```

Comment:
still trying to think of better category names.

illustrations, reactions -> used as `background-image`
nav, social, app, ui -> output as `icons.svg`